### ISSUE TITLE
extra characters after \ at the end of i command

### ISSUE THREAD

**Issue by laurenbenichou** [zsh-users/antigen#238]
Hey folks!

Run into this issue this am. 

`sed: 2: "/\{$/,/^\}/!{
          ...": extra characters after \ at the end of i command`

Here is my .zshrc file: 

```
source ~/antigen/antigen.zsh

	POWERLEVEL9K_MODE='awesome-patched'

	# Disable dir/git icons
	POWERLEVEL9K_HOME_ICON=''
	POWERLEVEL9K_HOME_SUB_ICON=''
	POWERLEVEL9K_FOLDER_ICON=''

	DISABLE_AUTO_TITLE="true"
	POWERLEVEL9K_SHORTEN_DIR_LENGTH=2

	POWERLEVEL9K_VCS_STAGED_ICON='\u00b1'
	POWERLEVEL9K_VCS_UNTRACKED_ICON='\u25CF'
	POWERLEVEL9K_VCS_UNSTAGED_ICON='\u00b1'
	POWERLEVEL9K_VCS_INCOMING_CHANGES_ICON='\u2193'
	POWERLEVEL9K_VCS_OUTGOING_CHANGES_ICON='\u2191'

	POWERLEVEL9K_VCS_MODIFIED_BACKGROUND='yellow'
	POWERLEVEL9K_VCS_UNTRACKED_BACKGROUND='yellow'
	#POWERLEVEL9K_VCS_UNTRACKED_ICON='?'

	POWERLEVEL9K_LEFT_PROMPT_ELEMENTS=(status os_icon dir context vcs)
	POWERLEVEL9K_RIGHT_PROMPT_ELEMENTS=(status time)

	POWERLEVEL9K_SHORTEN_STRATEGY="truncate_middle"

	POWERLEVEL9K_TIME_FORMAT="%D{%H:%M \uE868  %d.%m.%y}"

	POWERLEVEL9K_STATUS_VERBOSE=false
	export DEFAULT_USER="$USER"

	antigen use oh-my-zsh

	COMPLETION_WAITING_DOTS="true"
	DISABLE_CORRECTION="true"
	ZSH_HIGHLIGHT_HIGHLIGHTERS=(main brackets root)

	antigen bundle gitfast
	antigen bundle ssh-agent
	antigen bundle bower
	antigen bundle sublime
	antigen bundle gem
	antigen bundle node
	antigen bundle npm
	antigen bundle dirhistory
	antigen bundle jsontools
	antigen bundle github
	antigen bundle git-extras
	antigen bundle web-search
	antigen bundle osx
	antigen bundle themes
	antigen bundle grunt
	antigen bundle command-not-found
	antigen bundle chrissicool/zsh-256color
	antigen bundle caarlos0/zsh-add-upstream
	antigen bundle caarlos0/zsh-git-sync
	antigen bundle zsh-users/zsh-syntax-highlighting
	antigen bundle zsh-users/zsh-history-substring-search
	antigen bundle unixorn/autoupdate-antigen.zshplugin
	antigen theme bhilburn/powerlevel9k powerlevel9k
	antigen apply

        [[ -s $(brew --prefix)/etc/profile.d/autojump.sh ]] && . $(brew --   prefix)/etc/profile.d/autojump.sh

       export PATH=“/usr/local/bin:$PATH”

       alias gum='git submodule update'
       alias pushit='git push origin HEAD'
       alias ncache='sudo npm-cache install npm'

       export NVM_DIR="/Users/lbenicho/.nvm"
       [ -s "$NVM_DIR/nvm.sh" ] && . "$NVM_DIR/nvm.sh"  # This loads nvm
```

Any help would be appreciated.

**Comment by desyncr** [comment]
Hi @laurenbenichou This seems to be an issue with BSD sed. I'm taking a look at it.

Thanks for reporting.

**Comment by laurenbenichou** [comment]
Thank you!

**Comment by desyncr** [comment]
Should be fixed in next release [v1.1.3](https://github.com/zsh-users/antigen/milestone/6). Fix currently available in `develop`.

**Comment by laurenbenichou** [comment]
Works. Thanks!
